book.rb

```diff
@@ -0,0 +1,10 @@
+# Class representing a Book
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Please make sure to add a method in the Book class to add rentals.
  - You can consider using this code.
  
```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end